Reqnroll/ITestRunner.cs

```diff
@@ -4,16 +4,10 @@ namespace Reqnroll
 {
     public interface ITestRunner
     {
-        /// <summary>
-        /// The ID of the parallel test worker processing the current scenario. How the worker ID is obtained is dependent on the test execution framework.
-        /// </summary>
-        string TestWorkerId { get; }
         FeatureContext FeatureContext { get; }
         ScenarioContext ScenarioContext { get; }
         ITestThreadContext TestThreadContext { get; }
 
-        void InitializeTestRunner(string testWorkerId);
```

Comment:
No problem.
Are the Interface changes in `ITestRunnerManager` problematic? Because they are needed. Especially the `ReleaseTestThreadContext` method.

Reply:
No that cannot really be overridden in plugins. So that is more "internal".

Reply:
I added it back with an `Obsolete` attribute.
Should we add an issue for v3 to remember to remove it?

Reply:
Thx. I have added a task for it: https://github.com/orgs/reqnroll/projects/3?pane=issue&itemId=65349394